core/conn/trafci/src/org/trafodion/ci/Properties/trafciDefaultLookAndFeel.properties

```diff
@@ -1,3 +1,26 @@
+#
```

Comment:
I think the code is working fine with the copyright. The swjdbc tests and phoenix test ran fine.